floats_test.go

```diff
@@ -905,6 +905,16 @@ func TestProd(t *testing.T) {
 	}
 }
 
+func TestReverse(t *testing.T) {
+	s := []float64{9, 8, 7, 6, 5, 4, 3, 2, 1, 0}
```

Comment:
10 is an even number, but yes, I've added a small range 1-4 elements and the existing test.